.gitignore

```diff
@@ -14,3 +14,2180 @@
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/12.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/13.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/14.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/15.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/16.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/17.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/18.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/19.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/20.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/21.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/22.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/23.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/24.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/25.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/26.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/27.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/28.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/29.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/30.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/31.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/32.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/33.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/34.jpg
+sdk/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/data/odFridgeObjectsMask/images/35.jpg
```

Comment:
thanks for thinking to modify the .gitignore! perhaps can ignore folders so that don't have to list out files?